manager/driver/__init__.py

```diff
@@ -49,5 +49,5 @@ def upload(self, name, src_path, dst_path):
         pass
 
     @abstractmethod
-    def cleanup(self, image_prefix=""):
+    def cleanup(self, image_prefix="", versions=None):
```

Comment:
Is it possible to avoid adding this parameter? Perhaps using image tag instead of having the version in the name can simplify cleanup.